### ISSUE TITLE
October 2023: What future work do you recommend for testbeds, pilots, code sprints and other OGC initiatives?

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/developer-events#92]
Based on the outcomes of the October 2023 Code Sprint, what future work do you recommend for testbeds, pilots, code sprints and other OGC initiatives?

What work (e.g. new standards/profiles/best practices) would you like to recommend to the Standards Working Groups?

**Comment by tomkralidis** [comment]
OGC API - Records
- Extension for faceting
OGC API - Pub/Sub
- use case experimentation for various OGC API standards workflows
  - Records: aggregation/metadata ingest/publication
  - Processes: subscribing to async jobs channels
  - Collections: publishing data updaetes (e.g. publication of new NWP model run for EDR or Maps [cc @solson-nws @chris-little)
  - associating OGC API collections from with AsyncAPI (`x-ogc-api-collectionId`)

**Comment by chris-little** [comment]
@ghobona I agree with @tomkralidis . In particular, I think that the use of PubSub with Collections or a Collection is very important.

**Comment by IvanSanchez** [comment]
I personally would like a it more of SensorThings, and OGCAPI Tiles.

Also, multiband rasters in OGCAPI maps/tiles/coverages (by "multiband" I mean non-RGB and non-RGBA; either single-band elevation, or two-band slope, or 12-band Sentinel multispectral, etc etc). Including the questions: does it make sense to have it maps and tiles? How to symbolize multiband data?

It'd be very interesting to see a "GIMI vs GeoTIFF" shootout. I recall some GeoTIFF/COG people in a previous codesprint, so let's put them together in the same room with the NGA people in charge of GIMI.

**Comment by bradh** [comment]
> Also, multiband rasters in OGCAPI maps/tiles/coverages (by "multiband" I mean non-RGB and non-RGBA; either single-band elevation, or two-band slope, or 12-band Sentinel multispectral, etc etc). Including the questions: does it make sense to have it maps and tiles? How to symbolize multiband data?

Extension to this - tiled complex, signed, and floating point data. The viewpoint is where the tiles are a way for machines to process an area of interest (potentially in parallel), rather than as pictures to show to the user.

**Comment by chris-little** [comment]
@ghobona Perhaps a standard workflow is something like:
Observation->Collection->Exposing availability->Selection of data->Process->Generate visualisation (which may be maps or not, such as graphs)->Dissemination of results->Archiving.

Though Archiving may be applicable at any stage, and also PubSub, and Visualisation. And a lot is hidden in Process.

I suspect Q/C is applicable at every stage too, though the forms may be widely different.

**Comment by GeoCarsten** [comment]
Hi, The MUDDI SWG would like to continue with data encoding related work, though we are currently debating whether this is better done as part of a larger codesprint or standalone. Both approaches hae different pros and cons.
